akka-remote/src/main/scala/akka/remote/Endpoint.scala

```diff
@@ -809,6 +809,12 @@ private[remote] class EndpointWriter(
 
     case s: Send ⇒
       enqueueInBuffer(s)
+
+    case OutboundAck(_) ⇒
```

Comment:
yes, the case in `unhandled` is for all normal outgoing acks, this is just so that we ignore them during the take over process